x/leverage/keeper/borrows.go

```diff
@@ -94,15 +94,15 @@ func (k Keeper) GetBorrowerBorrows(ctx sdk.Context, borrowerAddr sdk.AccAddress)
 		var amount sdk.Int
 		if err := amount.Unmarshal(val); err != nil {
 			// improperly marshaled loan amount should never happen
-			return sdk.NewCoins(), err
+			panic(err)
```

Comment:
Is there a reason we panic here? Can we not bubble the error upstream?

EDIT: I guess this is OK and probably good since its an encoding error.